<?xml version="1.0"?>
<!-- Example launch file for stereo camera application -->
<launch>
  <group>
  <push_ros_namespace namespace="rgb_stereo"/>
    <!-- <node pkg="stereo_image_proc" name="disparity_node"  exec="disparity_node"/> -->
    <!-- <node pkg="stereo_image_proc" name="point_cloud_node" exec="point_cloud_node"/> -->

    <!-- Left -->
    <include file="$(find-pkg-share avt_vimba_camera)/launch/mono_camera.launch.xml">
      <arg name="params_file" value="$(find-pkg-share avt_vimba_camera)/config/phd.yaml"/>
      <arg name="name" value="left"/>
      <arg name="use_measurement_time" value="true" />
      <arg name="start_imaging" value="true" />
      <arg name="ip" value=""/>
      <arg name="guid" value="DEV_1AB22C0347E6"/>
      <arg name="camera_info_url" value="$(find-pkg-share avt_vimba_camera)/calibrations/calibration_example.yaml"/>
    </include>

    <!-- Right -->
    <include file="$(find-pkg-share avt_vimba_camera)/launch/mono_camera.launch.xml">
      <arg name="params_file" value="$(find-pkg-share avt_vimba_camera)/config/prof.yaml"/>
      <arg name="name" value="right"/>
      <arg name="use_measurement_time" value="true" />
      <arg name="start_imaging" value="true" />
      <arg name="ip" value=""/>
      <arg name="guid" value="DEV_1AB22C0347E7"/>
      <arg name="camera_info_url" value="$(find-pkg-share avt_vimba_camera)/calibrations/calibration_example.yaml"/>
    </include>
  </group>
</launch>
